### PR TITLE
Bootstrap backend w/ Docker

### PR DESCRIPTION
**Pull request by frm**

### CODE REVIEW

File: docs/insomnia.json [hunk 1]
```diff
@@ -0,0 +1 @@
...a1a38e44c46a571089f27ad2b93"}],"authentication":{},"metaSortKey":-1619018505208,"isPrivate":false,"settingStoreCookies":true,"settingSendCookies":true,"settingDisableRenderRequestBody":false,"settingEncodeUrl":true,"settingRebuildPath":true,"settingFollowRedirects":"global","_type":"request"},{"_id":"wrk_de6d8732a0d94d54922754259cc8831b","parentId":null,"modified":1619018490808,"created":1619018490808,"name":"Fractal","description":"","scope":null,"_type":"workspace"},{"_id":"env_783a90f56528b9d49eb9c7ed2d5281f3152cbc61","parentId":"wrk_de6d8732a0d94d54922754259cc8831b","modified":1619018490877,"created":1619018490877,"name":"Base Environment","data":{},"dataPropertyOrder":null,"color":null,"isPrivate":false,"metaSortKey":1619018490877,"_type":"environment"},{"_id":"jar_783a90f56528b9d49eb9c7ed2d5281f3152cbc61","parentId":"wrk_de6d8732a0d94d54922754259cc8831b","modified":1619018490880,"created":1619018490880,"name":"Default Jar","cookies":[],"_type":"cookie_jar"},{"_id":"spc_90ccb112b2ab4fba8907edbfffc4fb56","parentId":"wrk_de6d8732a0d94d54922754259cc8831b","modified":1619018490815,"created":1619018490815,"fileName":"Fractal","contents":"","contentType":"yaml","_type":"api_spec"}]}
```
Comment (djcouto) [inline docs/insomnia.json:1]: Is this file needed?

File: package.json [hunk 2]
```diff
@@ -1,9 +1,17 @@
...
     "contracts",
     "sdk",
     "types"
-  ]
+  ],
+  "scripts": {
+    "backend:start": "yarn workspace @fractalwallet/backend start",
+    "backend:build": "yarn workspace @fractalwallet/backend build",
+    "backend:dev": "yarn workspace @fractalwallet/backend dev",
+    "sdk:build": "yarn workspace @fractalwallet/sdk build"
```
Comment (djcouto) [inline package.json]: Please add the other builds script:

```
"types:build": "yarn workspace @fractalwallet/types build"
"plugin:build": "yarn workspace @fractalwallet/plugin build"
```

**Review by djcouto — APPROVED** [review]

File: docs/insomnia.json (hunk 1 above)
Comment (frm) [reply docs/insomnia.json:1]: It's not needed, it's an Insomnia workspace, to mimic the API requests. I usually include it to make sure everyone has the same requests. I can remove it if you prefer.

Comment (djcouto) [reply docs/insomnia.json:1]: No, I just thought that it was a forgotten file. 
Please let it stay and ignore my comment 👍
